cpp/examples/scripts/micro-osd.sh

```diff
@@ -0,0 +1,103 @@
+#
+#    Copyright (C) 2013,2014 Loic Dachary <loic@dachary.org>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
```

Comment:
We can't commit an AGPL-licensed script into our repo. If it's available somewhere else, we can just remove it from here and link to it from the docs instead?

Reply:
This is actually a modified version of https://github.com/ceph/ceph-rust/blob/master/micro-osd.sh. I added some commands to create a CephFS mount after creating a Ceph cluster. I don't really know a lot about licensing, but can we instead move this to ASF license and reference the original script ?

Reply:
No, if it's derived from an AGPL script it must still be AGPL. We can reference the file, but we can't hve this script here.